lunatrace/bsl/frontend/src/components/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright by LunaSec (owned by Refinery Labs, Inc)
+ *
+ * Licensed under the Business Source License v1.1
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ * https://github.com/lunasec-io/lunasec/blob/master/licenses/BSL-LunaTrace.txt
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import React from 'react';
+import { Button, Modal } from 'react-bootstrap';
+
+interface ConfirmationDialogProps {
+  onClose: (success: boolean) => void;
+  show: boolean;
+  title?: string;
+  body?: string;
+}
+
+export const ConfirmationDailog: React.FC<ConfirmationDialogProps> = ({ onClose, show, title, body }) => {
+  return (
+    <>
```

Comment:
I am curious -- what does this blank element do when you only have one child? Is it necessary here or is it extraneous?

Reply:
yes, definitely.  Theyre harmless though and I kind of like just having them there already when I want to render multiple elements from a component.  idk, im not sure if its harmful.